Tests/TSCUtilityTests/PlatformTests.swift

```diff
@@ -0,0 +1,53 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import XCTest
+import TSCBasic
+import TSCTestSupport
+
+import TSCUtility
+
+final class PlatformTests: XCTestCase {
+#if os(Linux)
```

Comment:
These tests can be run on all platforms so I suggest dropping this `#if`.

Reply:
The check executes `uname` to figure out the platform, so when executed on a mac, it will always return `.darwin`.

Reply:
Ah. Can you move the switch case into a different function so it is unit testable?

Reply:
Ok, will do.